### PR TITLE
Unknown users filter in reports

### PR DESCRIPTION
**Pull request by benrudolph**
:eyes:-team @calellowitz @kaapstorm i talked this over with @NoahCarnahan and we think that this will include some false positives in the reports, however in the next few days that will be corrected by a reindexing of the xforms es index. after discussing we decided that it makes sense to just wait for that reindex. once that goes through we can delete the last two commits on this PR and just have hte tests to make sure it works the same

http://manage.dimagi.com/default.asp?219739

### CODE REVIEW

File: corehq/apps/reports/standard/monitoring.py [hunk 1]
```diff
@@ -324,8 +331,24 @@ def es_queryset(self, users_by_id):
             .aggregation(active_total_aggregation)\
             .aggregation(inactive_total_aggregation)
 
-        query = case_es.CaseES().domain(self.domain).user(users_by_id.keys())
+
+        query = (
```
Comment (dimagimon) [inline corehq/apps/reports/standard/monitoring.py]: E303 too many blank lines (2)

**Comment by NoahCarnahan** [comment]
:+1: 
Is the "hold off on merge" tag intentional?

**Comment by benrudolph** [comment]
yeah cause once your fix is in, i don't think i'll need any of the commits except for the tests

**Comment by NoahCarnahan** [comment]
sounds good

**Comment by benrudolph** [comment]
@NoahCarnahan maybe we should just merge this and then revert the changes once your PR has gone through QA. that way we can stop the 500s for the time being?

**Comment by kaapstorm** [comment]
Sounds like a good plan to me.

**Pull request merged**